examples/crud-mcp/package.json

```diff
@@ -23,7 +23,7 @@
   },
   "dependencies": {
     "@modelcontextprotocol/sdk": "^1.9.0",
-    "@xava-labs/mcp": "file:../../packages/mcp",
```

Comment:
Lets leave the file reference so we can easily dev new features locally / test them? Or is this not a proper std?